fresnel/geometry.py

```diff
@@ -133,15 +143,25 @@ def __init__(self,
                  radius=0.5,
                  color=(0, 0, 0),
                  N=None,
-                 material=material.Material(solid=1.0, color=(1, 0, 1)),
```

Comment:
Yes, I plan to add this to the change log. Note that I don't update change logs in PRs to avoid merge conflicts. I write the change log in the PR description and then copy and paste it in after merging.

I also changed the default to solid=0. Now users will get an image closer to what they expect when using the default material.